tests/conftest.py

```diff
@@ -597,6 +597,9 @@ def invoices_for_sap_with_different_payment_method():
     get filtered out when generating summary reports"""
     invoices = [
         {
+            "contains_multibyte": [
```

Comment:
@hakbailey I added this multibyte element to this existing fixture and included it in the existing test for generating the summary. Should I make a separate fixture and test instead (invoice_that_includes_multibyte and test_generate_summary_with_multibyte)?

Reply:
Nah, I think this is fine.